qcengine/config.py

```diff
@@ -158,7 +158,7 @@ class TaskConfig(pydantic.BaseModel):
     scratch_messy: bool = pydantic.Field(
         False, description="Leave scratch directory and contents on disk after completion."
     )
-
+        
```

Comment:
Introducing the whitespace here probably caused a problem with our linter.

Could you run `make format` and issue a commit with the changed files?

Reply:
Done by https://github.com/MolSSI/QCEngine/pull/327/commits/d7ced823512ce58133739327684cae380592309c  😉